pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -320,22 +355,19 @@ abstract class Analytics {
 class AnalyticsImpl implements Analytics {
   final DashTool tool;
   final FileSystem fs;
-  late final ConfigHandler _configHandler;
+  final int toolsMessageVersion;
+  final ConfigHandler _configHandler;
   final GAClient _gaClient;
   final SurveyHandler _surveyHandler;
-  late String _clientId;
-  late final File _clientIdFile;
-  late final UserProperty userProperty;
-  late final LogHandler _logHandler;
-  late final Session _sessionHandler;
-  late final ErrorHandler _errorHandler;
-  final int toolsMessageVersion;
```

Comment:
Everything that is `late` has been either passed into `AnalyticsImpl` or created in the constructor's initializer list